source/includes/preview-warning-top.rst

```diff
@@ -0,0 +1,13 @@
+{+c2c-full-product-name+} is preview software.
+{+c2c-product-name+} should not be used in production systems.
+
+MongoDB does not guarantee that {+c2c-product-name+} will be
+released and reserves the right to modify any part of
+{+c2c-product-name+} without prior notice.
+
+MongoDB does not support unreleased software, however we encourage
+your feedback. If you encounter an error using {+c2c-product-name+},
+or if you have any suggestions about how we could improve
+{+c2c-product-name+}, please contact the MongoDB Product Management
+team.
+
```

Comment:
Similar to how other "Preview" features are mentioned in DOCs, please see https://docs-atlas-staging.mongodb.com/cloud-docs/docsworker-xlarge/DOCSP-22663/data-federation/query/query-with-sql/. 

Prefer the "blue" banner vs the "red". 

Below is the shortened text that I would like to use
`Cluster-to-Cluster Sync is available as a Preview feature. This feature and the corresponding documentation may change at any time during the Preview phase. Cluster-to-Cluster Sync should not be used in production systems.`